samples/csharp_dotnetcore/14.nlp-with-dispatch/README.md

```diff
@@ -159,7 +160,7 @@ Any time the bot file is encrypted, make sure to set the botFileSecret environme
 
```

Comment:
For the Visual Studio instructions, need to set the NLP-withDispatch project as your startup project before hitting F5.